src/lib/core/fiber.h

```diff
@@ -855,6 +875,73 @@ fiber_name(struct fiber *f)
 	return f->name;
 }
 
+/**
+ * Time since fiber was called.
+ */
+static inline double
+fiber_time_from_call(struct cord *cord)
+{
+	return clock_monotonic() - cord->call_time;
+}
+
+/**
+ * Check if fiber has custom slice.
+ */
+static inline bool
+fiber_has_custom_slice(struct fiber *fiber)
+{
+	return fiber->custom_slice.err != 0.0;
```

Comment:
IMO better add a fiber flag for this instead.

Reply:
Done